README.md

```diff
@@ -57,46 +58,36 @@ console.log('Listening on http://localhost:' + serve.address().port)
 
 Creates a HTTP server that serves entries from a `Hyperdrive` or `Localdrive`.
 
-`getDrive` is a required option. Given an id, it should return either a drive or null (indicating the drive is not available)
-
-```
-{
-  async getDrive (id) {
-    // return the drive
-  }
-}
-```
-
-Use the `drive` query param to select which drive to use, i.e. `/filename?drive=<id>`.
+Available query params:
+- `key` to select which drive to use i.e. `/filename?key=<id>`.
+- `version` to checkout into a specific point i.e. `/filename?version=<v>`.
 
 Available `options`:
 ```js
 {
-  async releaseDrive (id, drive) {
-    // release the drive if you want
-  },
-  async filter (id, filename, driveSnapshot) {
-    // if you dont want to serve this file on this drive,
-    // return false, or return true if you wanna
-  },
+  async get ({ key }) {}, // Return the drive or null
```

Comment:
can we also pass the other metadata here? like filename and version

Reply:
I also thought about using `get` as `filter` which it would require the args you say

But I noticed args added that weren't actually used, so better to keep args at minimal, but let's hold this until before merging the PR to see if we can delete the `filter` option thus adding the args you said